### ISSUE TITLE
feat: Patch for allowing Taking Screenshots in Telegram app.

### ISSUE THREAD

**Issue by gthu34** [revanced/revanced-patches#2565]
### Application

Telegram

### Issue

**Telegram** has a feature named "Restrict content"  which **disables taking screenshot and copying text in Channels,Groups and Bots** whether its private or pubic.
**It is NOT a server side thing** and doesnt go away even if you have Telegram premium subscription.
**This is a client side restriction which disables "taking scrrenshot"**. 

On a rooted phone with some [magisk module](https://github.com/Xposed-Modules-Repo/com.varuns2002.disable_flag_secure/releases)  it is easily bypassable ig but not for Non-rooted phone users who cant root their phones.

### Patch

The patch should be able to allow taking screenshot and if possible then make one for allowing copying text too.
Choose any of these open source Telegram clients to make the patch if making patch for official [Telegram](https://play.google.com/store/apps/details?id=org.telegram.messenger)  isnt possible for some reason.
1. [Telegram](https://telegram.org/android) (official but from their own site)
2. [Forkgram](https://f-droid.org/en/packages/org.forkgram.messenger/) (a third party open source client on F-droid)
3. [Plus Messanger](https://play.google.com/store/apps/details?id=org.telegram.plus) (a third party closed source client on play store)

### Motivation

**These are very annoying and troublesome restrictions for users who have to take screenshot to remember ,and keep track of things or copy text to search things.** 

**I am a student and whenever i stumbled upon some posts or comments i cant save it to read or remember it later**. I am forced to wrote it down or borrow another phone to  take the photo.
**Sometimes you had to search something which a post or comment mentioned like a book name or an obscure author name or an ISBN no**. since copying text is disable ,you have to type bit by bit which is again super annoying.
 I think it will be valubale to people alike  who face this problem.
 please drop a patch for allowing taking screenshts and also if possible then one for copying text from posts/comments too.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Reopen if nkt5

**Comment by gthu34** [comment]
No it doesnt work. 
**The Telegram just freeze on login screen** after patching and inThe Telegram just freeze on login screenstalling patched apk and unnstalling original one since it doesnt allow updating.
 Tried with Forkgram and login was sucessful but couldnt take screenshot. Still saying "cant take screenshot due to security policy" . I tried all three patches. 
1. Remove screen capture restriction.
2. Remove screenshot restriction.
3. Change package name
But nothing happened.
I am on android 11 if that helps.
